### ISSUE TITLE
Plugins - Success Message Not Announced by screen readers

### ISSUE THREAD

**Issue by kimisgold** [omeka/Omeka#1118]
Description: On the plugin page with an active screen reader, when the user clicks Activate, Deactivate, Install, Uninstall, or Configure, a success message appears at the top of the page (once the action is completed):
e.g., “The Coins plugin was successfully deactivated!”. However, this message is not announced, screen reader users are unaware of the status change.

Note: This also occurs when saving the 'Configure Theme' on the Appearance page

Expected Results: The success status messages should be announced automatically by assistive technologies without requiring user focus.

Actual Results: After updating the Plugin page by clicking a button, the success message appears visually at the top of the page. The message is not announced by screen readers.

(4.1.3 Status Messages; Level AA; WCAG 2.1)
